substrate/frame/child-bounties/src/migration.rs

```diff
@@ -0,0 +1,163 @@
+use super::*;
+// use codec::{Decode, Encode};
+use core::marker::PhantomData;
+use frame_support::{
+	storage_alias,
+	traits::{Get, UncheckedOnRuntimeUpgrade},
+};
+
+#[cfg(feature = "try-runtime")]
+use alloc::vec::Vec;
+#[cfg(feature = "try-runtime")]
+use frame_support::ensure;
+
+pub mod v1 {
+	use super::*;
+
+	pub struct MigrateToV1Impl<T>(PhantomData<T>);
+
+	#[storage_alias]
+	type ChildBountyDescriptions<T: Config + pallet_bounties::Config> = StorageMap<
+		Pallet<T>,
+		Twox64Concat,
+		BountyIndex,
+		BoundedVec<u8, <T as pallet_bounties::Config>::MaximumReasonLength>,
+	>;
+
+	#[storage_alias]
+	type ChildrenCuratorFees<T: Config> =
+		StorageMap<Pallet<T>, Twox64Concat, BountyIndex, BalanceOf<T>, ValueQuery>;
+
+	impl<T: Config> UncheckedOnRuntimeUpgrade for MigrateToV1Impl<T> {
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			// increment reads/writes after the action
```

Comment:
we need to skip this migration if storage version is not equal to `0`.
also please double check that the storage version will be updated to `1`. before we did it manually within the migration, but now may be it is done automatically by some type

Reply:
Do we need to deal with version checks manually? `VersionedMigration` type which we expose for this migration seems to do these checks internally accounting for the storage version.

This pallet didn't have storage version before this PR and it is introduced to `1` now.

Reply:
if there is already checks to make sure this migration wont be run on `0` version, than for sure no extra checks